connectors/src/connectors/notion/temporal/client.ts

```diff
@@ -60,11 +58,11 @@ export async function launchNotionSyncWorkflow(
         connectorId,
         startFromTs,
         forceResync,
-        garbageCollectionMode: useDualWorkflow ? "never" : "auto",
+        garbageCollectionMode: "never",
       },
     ],
     taskQueue: QUEUE_NAME,
-    workflowId: getNotionWorkflowId(dataSourceConfig),
+    workflowId: getNotionWorkflowId(dataSourceConfig, "never"),
```

Comment:
that's change no?

Reply:
(a change of behavior I mean)

Reply:
➕

Reply:
nope

Reply:
"Never" and "auto" generate the same workflow id